src/sql/azdata.d.ts

```diff
@@ -2187,6 +2187,10 @@ declare module 'azdata' {
 			 * What type of token this is (such as Bearer)
 			 */
 			tokenType?: string | undefined
+			/**
+			 * Access token expiry timestamp
+			 */
+			expiresOn?: number
```

Comment:
should this be in the proposed API changes, or we're pretty confident this is a long-term stable change?  basically once we put it in stable we shouldn't remove or change it.

Reply:
Whoops, good catch...yes @caohai please put this in proposed for now.

Reply:
done